packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -1417,6 +1417,57 @@ abstract class ModalRoute<T> extends TransitionRoute<T> with LocalHistoryRoute<T
     return child;
   }
 
+  /// The delegated transition provided to the previous route.
```

Comment:
Nit: Could you link this "delegated transition" text to [delegatedTransition] somewhere that has more information? Same for "received" below.

Reply:
Also, maybe mention "receivedTransition" here (and vice versa in "receivedTransition"'s docs). Something like:

```
/// The [delegatedTransition] provided to the [receivedTransition] of the route
/// below this one in the navigation stack.
```

And we should normalized on a way to refer to the "previous route", like maybe as I put it the "route below this one in the navigation stack" or something more elegant.